app/models/container.rb

```diff
@@ -4,5 +4,6 @@ class Container < ActiveRecord::Base
 
   belongs_to :container_group
   delegate   :ext_management_system, :to => :container_group
+  delegate   :container_project, :to => :container_group
```

Comment:
> @abonas this should be :through right? (Also above, but that would be another patch)

@moolitayer , 
I don't fully understand the scope of this patch - please update title and commit message to be more specific, "add relationships" is very general.

@simon3z I am not sure that "through" can be used here, because it is used with "has_X", while here we have "belongs_to" on container side. the "has_many through" can be used on the "other side" of the relationship.
But once @moolitayer will clarify this patch's scope, it will be easier to determine.